package.json

```diff
@@ -55,12 +55,12 @@
     "eslint": "^3.8.1",
     "eslint-plugin-react": "^6.4.1",
     "json-loader": "^0.5.4",
-    "leaflet": "^1.0.3",
-    "leaflet-draw": "^0.4.9",
+    "leaflet": "^1.3.1",
+    "leaflet-draw": "^1.0.2",
```

Comment:
I believe this version of leaflet-draw is broken:
https://github.com/Leaflet/Leaflet.draw/issues/872

However, a new 1.0.3 tag was released 6 days ago that fixes this. It should probably use that one instead.

Reply:
Updated to leaflet-draw@1.0.3